client/src/App.jsx

```diff
@@ -120,7 +120,7 @@ export class App extends React.Component {
                 />
               </Route>
               <Route path="/posts/:postID">
-                <PostPage />
+                <PostPage filteredCategories={this.state.filteredCategories} />
```

Comment:
Why are we filtering post page? Don't we risk filtering the results of the post you've actually clicked on from the map?